src/utils/__tests__/internal.ts

```diff
@@ -1291,7 +1293,7 @@ describe('assertExpectedChainVersion', () => {
     await signal;
     expect(warnSpy).toHaveBeenCalledTimes(1);
     expect(warnSpy).toHaveBeenCalledWith(
-      `This version of the SDK supports Polymesh chain spec version ${SUPPORTED_SPEC_VERSION_RANGE}. The chain spec is at version ${mockSpecVersion}. Please upgrade the SDK`
+      `This version of the SDK supports Polymesh chain spec versions "${SUPPORTED_SPEC_VERSION_RANGE}" and "${PRIVATE_SUPPORTED_SPEC_VERSION_RANGE}". The chain spec is at version ${mockSpecVersion}. Please upgrade the SDK`
```

Comment:
I feel this message will be a bit unclear when using mainnet. Is there a way to detect which version to use prior to this message?